insights/parsers/cron_jobs.py

```diff
@@ -0,0 +1,44 @@
+"""
+Cron job files under /etc/cron* but not users crontab
+=====================================================
+
+This module contains the following parsers:
+
+CronForeman - file ``/etc/cron.d/foreman``
+------------------------------------------
+"""
+
+from insights.core import Scannable
+from insights.core.plugins import parser
+from insights.specs import Specs
+
+
+@parser(Specs.cron_foreman)
+class CronForeman(Scannable):
```

Comment:
@huali027 why not use the [crontab](https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/crontab.py) parser here instead of Scannable, and add to the module with the other crontab data parsers?

Reply:
@bfahr, the jobs under "/ect/cron.d" is a little different from "crontab", this is from the man doc:
```
Cron checks these files and directories:

/etc/crontab
      system  crontab.   Nowadays  the  file  is empty by default.  Originally it was usually used to run daily, weekly, monthly jobs.  By default these jobs are now run through anacron which reads /etc/anacrontab configuration file.  See anacrontab(5) for more details.

/etc/cron.d/
     directory that contains system cronjobs stored for different users.

/var/spool/cron
    directory that contains user crontables created by the crontab command.
```

for the jobs in "/etc/cron.d/*", the format is like:
```
* * * * * root touch /tmp/file
```
Additionally, the `username` is extra needed in the jobs line. 

Since it is classified in the cron doc and also the format is a little different, I didn't put it in the "crontab" module. Currently, the "crontab" module covers the jobs in the "/var/spool/cron" directory.

And also the new advisor rule does not care about the time, just the command, so I use "Scannable" directly.
Do we need to put it together and enhance the `CrontabL` to support two kinds of formats?

Reply:
@huali027 thank you for the explanation.  What would probably be most helpful would be more examples of how to get the information from the parser.  A developer would really have to dig into the Scannable code to understand how this parser provides data without more examples.

Reply:
@bfahr, sure, I'll add more examples.